crates/fm/src/lib.rs

```diff
@@ -105,7 +105,10 @@ impl FileManager {
 ///
 /// Plucked from https://github.com/rust-lang/cargo/blob/fede83ccf973457de319ba6fa0e36ead454d2e20/src/cargo/util/paths.rs#L61
 /// Advice from https://www.reddit.com/r/rust/comments/hkkquy/comment/fwtw53s/
-fn normalize_path(path: &Path) -> PathBuf {
+// TODO: Are we fine with this being a public API in fm?
+// Maybe it should be called `resolve_path` on the FileManager struct and appends the root if the path is relative?
+// We probably should rename the current `resolve_path` to `resolve_module` or something to be more clear on what it does
+pub fn normalize_path(path: &Path) -> PathBuf {
```

Comment:
Resolve TODO.

Reply:
I think this needs to be done in a separate PR. I can prepare a prereq PR.

Reply:
PR at #2156 and I'll rebase this once it is merged.